python/foglamp/services/core/api/service.py

```diff
@@ -123,8 +123,7 @@ def get_service_records():
         except ImportError as ex:
             # Checking for C-type plugins
             script = '["services/south_c"]' if service_type == 'south' else '["services/north_c"]'
-            plugin_info = apiutils.get_plugin_info(plugin)
-            plugin_config = plugin_info['config']
+            plugin_config = apiutils.get_plugin_info(plugin)
```

Comment:
Please revert this change

Reply:
Without this change the adding of new C services fails because there is no 'config' element in the JSON returned from get_plugin_info

Reply:
config is there,

O/P

plugins/utils/get_plugin_info plugins/south/dummy/libdummy.so plugin_info | jq
```
{
  "name": "Dummy",
  "version": "1.0.0",
  "type": "south",
  "interface": "1.0.0",
  "config": {
    "plugin": {
      "description": "Dummy C south plugin",
      "type": "string",
      "default": "dummy"
    }
  }
}
```
you may also look on https://github.com/foglamp/FogLAMP/blob/develop/C/plugins/utils/get_plugin_info.cpp#L55

Reply:
Looks like a version issue, previously in develop the config element was missing